nomad-aws/variables.tf

```diff
@@ -91,8 +91,20 @@ variable "instance_tags" {
   }
 }
 
-variable "nomad_auto_scaler" {
-  type        = bool
-  default     = false
-  description = "If true, terraform will create user access keys to be used by nomad autoscaler."
+# Check for IRSA Role (more details)  - https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html
+#   enable_irsa  = {
+#                  oidc_principal_id  = "arn:aws:iam::ACCOUNT_ID:oidc-provider/oidc.eks.REGION.amazonaws.com/id/OIDC_ID"
```

Comment:
nitpick: i think its easier to identify variables in strings when they are enclosed in <>
eg: "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/oidc.eks.<REGION>.amazonaws.com/id/<OIDC_ID>"
It's a bit more readable.